frameworks/AutoGluon/__init__.py

```diff
@@ -11,15 +11,12 @@ def run(dataset: Dataset, config: TaskConfig):
     from frameworks.shared.caller import run_in_venv
 
     data = dict(
-        train=dict(data=dataset.train.data),
-        test=dict(data=dataset.test.data),
+        train=dict(path=dataset.train.data_path('parquet')),
+        test=dict(path=dataset.test.data_path('parquet')),
```

Comment:
@Innixma here is the switch to parquet files.
The conversion is imply using `pandas.to_parquet` on the original dataframe, and the later doesn't stay in memory.

Reply:
looks good to me!